samples/net/cloud/google_iot_mqtt/src/protocol.c

```diff
@@ -77,11 +77,9 @@ static sec_tag_t m_sec_tags[] = {
  */
 time_t my_k_time(time_t *ptr)
 {
-	int64_t stamp;
 	time_t now;
 
-	stamp = k_uptime_get();
-	now = (time_t)((stamp + time_base) / 1000);
+	now = (time_t)((time_base + k_uptime_get()) / 1000);
```

Comment:
Changes in this file do not match your commit description. It is much better to separate such changes from the main thing you are doing ("use sntp_simple"), because it makes review faster and makes traversing git history much smoother.

Reply:
reverted